### PR TITLE
Provisional changes for TensorFlow v2

### PR DESCRIPTION
**Pull request by sglvladi**

### CODE REVIEW

File: docs/source/install.rst [hunk 1]
```diff
@@ -485,49 +380,96 @@ The ``pycocotools`` package should be installed if you are interested in using C
...
 
-Test your Installation
-~~~~~~~~~~~~~~~~~~~~~~
+Install the Object Detection API
+********************************
+Installation of the Object Detection API is achieved by installing the ``object_detection`` package. This is done by running the following commands from within ``Tensorflow\models\research``:
 
-- Open a new `Terminal` window and activate the `tensorflow_gpu` environment (if you have not done so already)
-- ``cd`` into ``TensorFlow\models\research\object_detection`` and run the following command:
+.. code-block::
```
Comment (mgalgs) [inline docs/source/install.rst:389]: FYI, this code block isn't rendering in the final html output... I don't know enough rst to troubleshoot :grimacing: 

image

File: docs/source/install.rst [hunk 2]
```diff
@@ -485,49 +380,96 @@ The ``pycocotools`` package should be installed if you are interested in using C
...
-       :alt: alternate text
-       :align: center
+.. _test_tf_models:
+
+Test your Installation
+**********************
+
+To test the installation, run the following command from within ``Tensorflow\models\research``:
 
+.. code-block::
```
Comment (mgalgs) [inline docs/source/install.rst:432]: oh this one isn't rendering either. Maybe they just need a language specifier (`bash`)?

Comment (sglvladi) [reply docs/source/install.rst:432]: Ahh, thanks for spotting that, I'll have a look in a minute

**Comment by sglvladi** [comment]
@mgalgs Should be fixed now (see [here](https://tensorflow-object-detection-api-tutorial.readthedocs.io/en/latest/training.html)).  Make sure to clear your browser cache, otherwise your browser may show a cached version of the page(s).

I found a few other instances that were not rendered correctly and fixed them as well. Turns out the ``code-block`` directive expects the specifier. Not sure why it was rendered correctly locally.

**Comment by mgalgs** [comment]
@sglvladi lgtm! Thanks again!

Training my first tf2 model right now... :tada:

**Pull request merged**